### PR TITLE
fix(test):C# example ineffective in CI under TDegnine/example

### PR DESCRIPTION
**Pull request by xleili**
移除冗余、失效C# CI test 项，源测试文件在 src/connector/C#/src/test, 因为已经将C# connector 已在独立仓库。
移除冗余、失效C# CI Cases项，源文件在 src/connector/C#/src/test/cases,因为该项已经移动到C# 独立仓库，改造为 src/Driver.Test/FunctionTest项

### CODE REVIEW

File: examples/C#/C#checker/C#checker.cs [hunk 1]
```diff
@@ -389,7 +389,7 @@ public void CloseConnection()
 
         static void ExitProgram()
         {
-            System.Environment.Exit(0);
+            System.Environment.Exit(1);
```
Comment (zitsen) [inline examples/C#/C#checker/C#checker.cs:392]: So it is for abnormally exit only?

File: examples/C#/C#checker/C#checker.csproj [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFramework>net5.0</TargetFramework>
+    <TargetFramework>net6.0</TargetFramework>
```
Comment (zitsen) [inline examples/C#/C#checker/C#checker.csproj:5]: Keep bounding with an older version if ok.

File: examples/C#/TDengineTest/TDengineTest.csproj [hunk 3]
```diff
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFramework>net5.0</TargetFramework>
+    <TargetFramework>net6.0</TargetFramework>
```
Comment (zitsen) [inline examples/C#/TDengineTest/TDengineTest.csproj:5]: Same.

File: examples/C#/stmt/StmtDemo.cs [hunk 4]
```diff
@@ -543,7 +543,7 @@ public void DisplayRes(IntPtr res)
         public static void ExitProgram()
         {
             TDengine.Cleanup();
-            System.Environment.Exit(0);
+            System.Environment.Exit(1);
```
Comment (zitsen) [inline examples/C#/stmt/StmtDemo.cs:546]: And here

**Review by zitsen — COMMENTED** [review]
See my comments

File: examples/C#/C#checker/C#checker.cs (hunk 1 above)
Comment (xleili) [reply examples/C#/C#checker/C#checker.cs:392]: yes,only for exceptional aborting.

File: examples/C#/C#checker/C#checker.csproj (hunk 2 above)
Comment (xleili) [reply examples/C#/C#checker/C#checker.csproj:5]: only .Net 6.0 is installed on CI ENV.

File: examples/C#/stmt/StmtDemo.cs (hunk 4 above)
Comment (xleili) [reply examples/C#/stmt/StmtDemo.cs:546]: same,only for exceptional aborting.

File: examples/C#/TDengineTest/TDengineTest.csproj (hunk 3 above)
Comment (xleili) [reply examples/C#/TDengineTest/TDengineTest.csproj:5]: same，only for exceptional aborting.

File: tests/parallel_test/cases.task [hunk 5]
```diff
@@ -236,6 +236,7 @@
 30,,script,./test.sh -f general/import/commit.sim 
 30,,script,./test.sh -f general/compute/diff2.sim 
 30,,develop-test,bash 3-connectors/R/test.sh
+30,,develop-test,bash 3-connectors/c#/test.sh
```
Comment (sangshuduo) [inline tests/parallel_test/cases.task:239]: @gccgdb1234  not sure if the C# connector examples need to be added in regular CI if the test consumes a few seconds or longer. And if other connectors' need to be added too if C# should

**Review by sangshuduo — APPROVED** [review]
LGTM but still wonder if the connector example should add to regular CI

**Pull request merged**